notebooks/implicit_differentiation.ipynb

```diff
@@ -0,0 +1,561 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "protecting-marine",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax-md/blob/main/notebooks/implicit_differentiation.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "official-indianapolis",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title Import & Util\n",
+    "!pip install -q git+https://www.github.com/google/jax\n",
+    "!pip install -q git+https://www.github.com/google/jax-md\n",
+    "!pip install jaxopt\n",
+    "\n",
+    "import jax\n",
+    "import jax.numpy as jnp\n",
+    "from jax.config import config\n",
+    "config.update('jax_enable_x64', True)\n",
+    "\n",
+    "from jax import random, jit, lax\n",
+    "\n",
+    "from jax_md import space, energy, minimize, quantity\n",
+    "\n",
+    "import jaxopt\n",
+    "\n",
+    "f32 = jnp.float32\n",
+    "f64 = jnp.float64"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "going-motion",
+   "metadata": {},
+   "source": [
+    "# Energy minimization functions"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Maybe this cell also be a form with the code hidden by default? Could also be merged with Import & Util above?

Reply:
Done